pages/account/manage/event/[[...data]].js

```diff
@@ -89,51 +90,51 @@ export default function ManageEvent({ BASE_URL, event }) {
       putEvent = { ...putEvent, _id: event._id };
       apiUrl = `${BASE_URL}/api/account/manage/event/${event._id}`;
     }
-    const res = await fetch(apiUrl, {
-      method: "PUT",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify(putEvent),
-    });
-    const update = await res.json();
+    try {
+      await sendRequest({
+        url: apiUrl,
+        method: "PUT",
+        headers: {
```

Comment:
I think you can move header `Content-type` to sendRequest util as default header

Reply:
Good idea 👍